client/me/purchases/vat-info/index.tsx

```diff
@@ -84,11 +100,58 @@ export default function VatInfoPage() {
 					</CardHeading>
 					<p className="vat-info__sidebar-paragraph">
 						{ translate(
-							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST"). */
-							"We currently only provide %(taxName)s invoices to users who are properly registered. %(taxName)s information saved on this page will be applied to all of your account's receipts.",
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							'The %(taxName)s details saved on this page will be applied to all receipts in your account.',
 							{
-								textOnly: true,
-								args: { taxName: taxName ?? translate( 'VAT', { textOnly: true } ) },
+								args: { taxName: taxName ?? translate( 'tax (VAT/GST/CT)', { textOnly: true } ) },
+							}
+						) }
+						<br />
+						<br />
+						{ translate(
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							/* translators: This is a list of tax-related reasons a customer might need to contact support */
+							'If you:' +
+								'{{ul}}' +
+								'{{li}}Need to update existing Tax ID details{{/li}}' +
+								'{{li}}Have been charged VAT as a VAT-Registered Business subject to reverse charges{{/li}}' +
+								'{{li}}Do not see your country listed in this form{{/li}}' +
+								'{{/ul}}' +
+								'{{contactSupportLink}}Contact our Happiness Engineers{{/contactSupportLink}}. Include your %(taxName)s number and country code when you contact us.',
+							{
+								args: { taxName: taxName ?? translate( 'tax', { textOnly: true } ) },
```

Comment:
I don't mind it but is there a reason to use `tax` as the fallback here instead of `tax (VAT/GST/CT)` like other places?

Reply:
At one point, I had replaced the first `tax (VAT/GST/CT)` with just `tax` because it looked strange to have it directly under the card title. I must have copied this over from that. 

I have changed all of the translation strings to `fallbackTaxName`. This way we only have to translate it once.